src/plugins/validate-semantic/validators/form-data.js

```diff
@@ -49,41 +49,43 @@ export const validateParameterFormDataForFileTypes = () => system => {
 
 export const validateParameterFormDataConsumesType = () => system => {
   return system.validateSelectors
-    .allParameters()
+    .allPathItems()
     .then(nodes => {
       return nodes.reduce((acc, node) => {
         const value = node.node
+        const globalConsumes = getRootNode(node).node.consumes ||[] 
+        var parameters = value.parameters
 
-        const globalConsumes = getRootNode(node).node.consumes ||[]
+        var isThereAnyFormData = parameters != null && parameters.find(parameter => parameter.in === "formData")
+        var isThereAnyFile = parameters != null && parameters.find(parameter => parameter.type === "file")
```

Comment:
**Change request**: change these from `vars to `const`s, since we're never reassigning their values — this is [a best practice](https://medium.com/dailyjs/use-const-and-make-your-javascript-code-better-aac4f3786ca1)!

Reply:
Done!